cmake/ActsCompilerOptions.cmake

```diff
@@ -13,6 +13,18 @@ set(cxx_flags
     "-Wall -Wextra -Wpedantic -Wshadow -Wzero-as-null-pointer-constant -Wold-style-cast"
 )
 
+# Add assertions to standard libraries for debug builds
+if(
+    (CMAKE_BUILD_TYPE STREQUAL RelWithDebInfo)
+    OR (CMAKE_BUILD_TYPE STREQUAL Debug)
+)
+    if(CMAKE_CXX_COMPILER_ID MATCHES "GNU")
+        set(cxx_flags "${cxx_flags} -D_GLIBCXX_ASSERTIONS")
+    elseif(CMAKE_CXX_COMPILER_ID MATCHES "Clang")
```

Comment:
I don't think clang implies libc++. I *think* on many linux distros clang compiles with libstdc++.

Reply:
right, I just realized this. As written below, I would add both flags unconditionally, because they should not hurt if the other stdlib is used I guess...